pkg/backup/volume_snapshot_action.go

```diff
@@ -59,6 +59,14 @@ func (a *volumeSnapshotAction) Execute(ctx ActionContext, volume map[string]inte
 
 	metadata := volume["metadata"].(map[string]interface{})
 	name := metadata["name"].(string)
+	labels := metadata["labels"]
```

Comment:
There are some helpers for navigating these maps in pkg/util/collections. You can get the labels map with `labelsMap, err := collections.GetMap(metadata, "labels")`. Unfortunately, you can't go straight to the zone right now because our helpers take dot-separated paths, which won't work given this label, but this should at least simplify getting the labels map.

Reply:
Will do. That should simplify things a little bit.